script.js

```diff
@@ -0,0 +1,24 @@
+var GalaxyS6 = new Telefon('Samsung', 2200, 'czarny','', '10', '');
+var iPhone6S = new Telefon('Apple', 3300, 'srebrny','USD', '10', '');
+var OnePlusOne = new Telefon('OnePlus', 1100, 'bialy','EUR', '20', '');
+
+function Telefon(brand, price, color, currency, warrantyPercent, warrantyCost) {
+	this.brand = brand;
+	this.price = price;
+	this.color = color;
+	this.currency = currency || 'PLN';
+	this.warrantyCost = getWarrantyCost(price, warrantyPercent);
+	this.warrantyPercent = warrantyPercent;
+}
+
+function getWarrantyCost(price, warrantyPercent) {	
+	return (warrantyPercent/100) * price;
+}
```

Comment:
Jest ok, ale w tym momencie konstruktor korzysta z funkcji, która leży sobie luźno w tym pliku. Jak jej zabraknie to będzie klops. Przenieś tą funkcję do prototypu (tak jak `printInfo`)